bitnami/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -237,4 +237,9 @@ spec:
         resources:
           requests:
             storage: {{ .Values.master.persistence.size | quote }}
+          {{- if .Values.master.persistence.existingVolume }}
+          selector:
+            matchLabels:
+              volume: {{ .Values.master.persistence.existingVoume }}
```

Comment:
Typo here

```suggestion
              volume: {{ .Values.master.persistence.existingVolume }}
```

Reply:
I don't think extraVolumes would cause the current volumeClaimTemplate to be ignored, and I think mounting an extra volume at exactly the same name and mountPath would be confusing if it did work, and I wouldn't want to depend upon such behavior. But, more importantly, there are 2 volumeClaimTemplate(s), one defined for .data. and one for .master., but extraVolumes is defined only at the root level (values.exrtaVolumes), so even if it could swap into place seamlessly, I'd have to define 'name: data' twice at two different mountPath(s)? I think the small addition in this request is cleaner and more comprehensible. Not that I'm a kubernetes expert at this point, so I'm open if you think that analysis is incorrect.

Reply:
Also, corrected that typo, thank you for catching that.